CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 ### Bug Fixes
 
 - Fixed Issue #831: Contract Checker Bug. Now raises `AssertionError` when the expected type is `float` but got `int` instead.
+- Now raises `AssertionError` when the expected type is `int` but got `bool` instead.
```

Comment:
You need to mention contract checking here, otherwise it's not clear what the subject of "raises" is. (This was implied by include the issue title in the previous entry.)